tpgtools/samples/assuredworkloads/workload/workload_post_destory.go

```diff
@@ -0,0 +1,78 @@
+// deleteAssuredWorkloadProvisionedResources deletes the resources provisioned by
+// assured workloads.. this is needed in order to delete the parent resource
+func deleteAssuredWorkloadProvisionedResources(t *testing.T) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		config := googleProviderConfig(t)
+		timeout := *schema.DefaultTimeout(4 * time.Minute)
+		for _, rs := range s.RootModule().Resources {
+			if rs.Type != "google_assured_workloads_workload" {
+				continue
+			}
+			resourceAttributes := rs.Primary.Attributes
+			n, err := strconv.Atoi(resourceAttributes["resources.#"])
+			log.Printf("[DEBUG]found %v resources\n", n)
```

Comment:
Do we want these log statements for the "happy" path?

Reply:
I think this is relevant. It's debug logs which already contain the rest responses. I think this makes the data more readable :)